docs/predictive-analytics/next-best-offer/index.md

```diff
@@ -0,0 +1,214 @@
+---
+seo:
+  title: Next Best Offer — Anticipating Your Customer's Needs 
+  description: This recipe demonstrates how to use ksqlDB to present relevant offers to banking customers.
+---
+
+# Next Best Offer — Anticipating Your Customer's Needs
+
+Consumers today are faced with never-ending marketing messages from a variety of sources. Often these messages are generic and don't have any consideration for the individual needs of the consumer. This one-size-fits-all approach leads to poor conversion rates. A better approach is to tailor offerings that take into consideration the interests of the consumer, based on previous purchases or behavior. This recipe demonstrates how to take existing customer information and provide a "Next Best Offer" to encourage sales and retain customers.
+
+## Step-by-step
+
+### Set up your environment
+
+Provision a Kafka cluster in [Confluent Cloud](https://www.confluent.io/confluent-cloud/tryfree/?utm_source=github&utm_medium=ksqldb_recipes&utm_campaign=next_best_effort).
+
+
+--8<-- "docs/shared/ccloud_setup.md"
+
+### Read the data in
+
+--8<-- "docs/shared/connect.md"
+
+```json
+--8<-- "docs/predictive-analytics/next-best-offer/source.json"
+```
+
+--8<-- "docs/shared/manual_insert.md"
+
+### Run the stream processing application
+
+This application will perform a series of joins between event streams and tables to calculate the next best offer for a banking consumer, based on their activity. This should yield higher customer activity and satisfaction.
```

Comment:
Since `consumer` is overloaded with Kafka and since the ksqlDB commands say `customer`, can this recipe `:%s/consumer/customer/g`?